javascript/chronometer.js

```diff
@@ -1,26 +1,52 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+   this.currentTime = 0;
+   this.intervalId = 0;
+
   }
+
   startClick(callback) {
-    // ... your code goes here
-  }
+    this.intervalId = setInterval(() => {
+      this.currentTime ++;
+      if (callback) {
+        callback();
+      }
+    }, 1000)
+  }
+
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60)
   }
+
   getSeconds() {
-    // ... your code goes here
-  }
-  twoDigitsNumber() {
-    // ... your code goes here
-  }
+    return this.currentTime % 60
+  }
+
+  twoDigitsNumber(number) {
+    console.log(number);
+    if (number < 10) {
+      return `0${number}`
+    } else {
+      return `${number}`;
+    }
```

Comment:
**ES6 tip**: A nice thing you could do here is to make use of the ternary operator. It's especially helpful when you have an ```if ...else``` statement that only does a singular thing within each code block. Check out the suggestion below and the video I've linked to get a better understanding of what I mean

```suggestion
    return number < 10 ? `0${number}` : `${number}`;
```

Find video here --> https://www.youtube.com/watch?v=70GmtV-5Ggo&ab_channel=WebDevSimplified